context/app/static/js/shared-styles/sections/Section.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import { headerHeight } from 'js/components/Header/HeaderAppBar/style';
+
+const baseOffset = headerHeight + 10;
+/* Anchor offset for fixed header.
+Only to be used on pages with table of contents. */
+const Section = styled.section`
+  margin-bottom: ${(props) => props.theme.spacing(5)}px;
+  scroll-margin-top: ${baseOffset}px;
+`;
+
+export { baseOffset };
+export default Section;
```

Comment:
We don't expect named exports from files with pascal case filenames, but I want to reuse the `baseOffset` in other styles like I do in this PR (context/app/static/js/components/detailPage/style.js) for detail page sections to handle the extra height from the entity header. Do you have a suggestion for how I could better share the `baseOffset`? Theme?

Reply:
Theme makes sense, but I'm ok with a rare non-default export, if the alternatives seem more awkward.